googledrive/googledrive_class_lib.py

```diff
@@ -0,0 +1,65 @@
+"""
+Модуль для работы с Google Sheets
+"""
+import googleapiclient.discovery  # pip install google-api-python-client
+import httplib2  # pip install httplib2
+from oauth2client.service_account import ServiceAccountCredentials
+import googledrive.googlesheets_vars as gs_vars
+import os
+
+# pip install google-api-python-client библиотека для работы с googlу api
+# pip install oauth2client библиотека для авторизации
+
+
+class GoogleSheets(object):
```

Comment:
В python3 не нужно передавать object

```suggestion
class GoogleSheets:
```

Reply:
ok